js/webpack.config.js

```diff
@@ -87,7 +87,7 @@ module.exports = {
         loader: 'file-loader'
       },
       {
-        test: /\.ico$/,
+        test: /parity-logo-black-no-text\.png$/,
```

Comment:
Ok, fair enough, but we could actually stick to the `name.extension` format for other images, or I think pass an argument while importing the image in the JS